### PR TITLE
Feature/tiling statement

### PR DESCRIPTION
**Pull request by Adamws33**
updated tiling option in soil drainage to show what is selected when tiling is active and show a statement for what the new drainage is while maintaining the calculations.

### CODE REVIEW

File: src/pages/Location/SoilCondition/SoilDrainage/RenderDrainageClasses.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ const RenderDrainageClasses = ({ tilingCheck, drainage = [] }) => {
   // redux vars
   const soilDataRedux = useSelector((stateRedux) => stateRedux.soilData.soilData);
   const councilShorthandRedux = useSelector((stateRedux) => stateRedux.mapData.councilShorthand);
-  const [previousDrainage, setPreviousDrainage] = useState(0);
+  const [previousDrainage, setPreviousDrainage] = useState(-1);
+  const [updateTileCheck, setUpdateTileCheck] = useState(true);
```
Comment (mikahpinegar) [inline src/pages/Location/SoilCondition/SoilDrainage/RenderDrainageClasses.js:19]: should this be updated to updatedTilingCheck, setUpdatedTilingCheck to be consistent with the other

Comment (mikahpinegar) [reply src/pages/Location/SoilCondition/SoilDrainage/RenderDrainageClasses.js:19]: also what's the difference between updateTileCheck and tilingCheck?

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
if i choose nc -> somewhat poorly drained -> yes to tiling it takes the drainage class down to poorly drained when it should go to moderately well drained

**Comment by mikahpinegar** [comment]
if i chose ny selecting tiling doesn't change anything and says my improved drainage class is the same as unimproved drainage class

**Comment by mikahpinegar** [comment]
in midwest if i select tile drainage it doesn't improve my drainage class it stays the same. unselecting reduces the selected chip

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by YazidZidane** [comment]
> if i chose ny selecting tiling doesn't change anything and says my improved drainage class is the same as unimproved drainage class

I'm also having same issue here. When I select NC -> Drainage Class: Very poorly drained -> Tile Drainage: true -> And it shows: Your improved drainage class is Very poorly drained.

Sometime it works and update drainage class to next level, but most time it just shows the same, especially first time enter the page.

**Comment by Adamws33** [comment]
> > if i chose ny selecting tiling doesn't change anything and says my improved drainage class is the same as unimproved drainage class
> 
> I'm also having same issue here. When I select NC -> Drainage Class: Very poorly drained -> Tile Drainage: true -> And it shows: Your improved drainage class is Very poorly drained.
> 
> Sometime it works and update drainage class to next level, but most time it just shows the same, especially first time enter the page.

In the updateDrainageClass function (line ~66 RenderDrainageClasses) i intended to move setUpdateTilingCheck into the if statement and also left that state update outside of the if.  This is updated now.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.253.112.197:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by mikahpinegar — APPROVED** [review]

**Pull request merged**